packages/components/pie-card-container/src/index.ts

```diff
@@ -31,19 +31,23 @@ export class PieCardContainer extends LitElement implements CardContainerProps {
     @property({ type: Object })
     public aria!: AriaProps;
 
+    @property({ type: Boolean })
+    public isDraggable = false;
+
     render () {
         const {
             href,
             target,
             rel,
             disabled,
             variant,
+            isDraggable,
         } = this;
 
         return html`
                 <div
                     variant=${variant}
-                    class="c-card-container"
+                    class="c-card-container ${isDraggable ? 'isDraggable' : ''}"
```

Comment:
can you please ensure this is a a separate line? 

and boolean values can be written as `? isDraggable =${isDraggable}` similar to the disabled state, this will ensure we are consistent with other implementations  and only add the prop if it's truthy